### PR TITLE
Make MFA explanation more user-oriented

### PR DESCRIPTION
**Pull request by stevenvdb**
This PR aims to make the MFA page clearer from the user's point of view.

### CODE REVIEW

File: source/leuven/tier2_hardware/mfa_login.rst [hunk 1]
```diff
@@ -1,143 +1,244 @@
...
-- copy/paste this URL in your browser and use the VSC account authentication
-
-Your SSH-connection will now be completed. This method also works from a Linux or Mac terminal.
-
-UI applications with SSH connection in the background
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+- Access to a VSC-associated university/institution account
+
+This document provides two methods to connect to the KU Leuven clusters with
+MFA. The first method, without an agent, is easier to get started but requires
```
Comment (w-lampaert) [inline source/leuven/tier2_hardware/mfa_login.rst:17]: to get started 'with'

File: source/leuven/tier2_hardware/mfa_login.rst [hunk 2]
```diff
@@ -1,143 +1,244 @@
...
+some repetitive steps with each connection. The second method, *with* an agent,
+requires some more effort to set up initially but will be easier when you need
+multiple connections or you are using some specific GUI applications to connect.
```
Comment (w-lampaert) [inline source/leuven/tier2_hardware/mfa_login.rst:20]: Would it be an idea to mention 'easier when you need... or **even necessary when** you are using some specific...'

File: source/leuven/tier2_hardware/mfa_login.rst [hunk 3]
```diff
@@ -1,143 +1,244 @@
...
+  the UI of an agent, where you specifically ask to create a certificate. You
+  will be redirected to the firewall link to verify your identity (vscagent).
+
+.. note ::
+
+   We provide an in-house developed `vscagent` as a temporary solution to have
+   an ssh agent on Windows that supports storing a certificate. We expect that
+   Pageant will support this as well in their future official release. Once we
+   can confirm this support, we will adapt our official recommendation to
+   Pageant instead of vscagent. This documentation will be adapted acordingly
```
Comment (w-lampaert) [inline source/leuven/tier2_hardware/mfa_login.rst:108]: accordingly

**Comment by stevenvdb** [comment]
Thanks; normally I try to respect read-only Friday, but if you are happy with the latest changes I will merge it today so the page is visible before MFA rolls out for all our users.

**Pull request merged**